sql/table_test.go

```diff
@@ -170,8 +188,21 @@ func TestMakeTableDescIndexes(t *testing.T) {
 		if err != nil {
 			t.Fatalf("%d: %v", i, err)
 		}
-		if !reflect.DeepEqual(d.index, schema.Indexes[0]) {
-			t.Fatalf("%d: expected %+v, but got %+v", i, d.index, schema.Indexes[0])
+		if !reflect.DeepEqual(d.primary, schema.PrimaryIndex) {
+			t.Fatalf("%d: expected %+v, but got %+v", i, d.primary, schema.PrimaryIndex)
+		}
+		if len(d.indexes) > 0 && !reflect.DeepEqual(d.indexes, schema.Indexes) {
+			t.Fatalf("%d: expected %+v, but got %+v", i, d.indexes, schema.Indexes)
 		}
+
+	}
+}
+
+func TestErrPrimaryKeyUnspecified(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	stmt, _ := parser.Parse("CREATE TABLE test (a INT, b INT, CONSTRAINT c UNIQUE (b))")
```

Comment:
I disagree; the savings are not worth it, except for cases when we _really_ know the error won't be triggered per external docs.